lib/components/block-link/block-link.visual.test.ts

```diff
@@ -0,0 +1,61 @@
+import { html } from "@open-wc/testing";
+import { defaultOptions, runComponentTests } from "../../test/test-utils";
+import "../../index";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const blockLinkTemplate = ({ component, testid }: any) => html`<div
+    class="d-inline-flex ai-center jc-center hs1 ws2 p8"
+    data-testid="${testid}"
+>
+    ${component}
+</div>`;
+
+describe("block-link", () => {
+    // Base block link
+    runComponentTests({
+        type: "visual",
+        baseClass: "s-block-link",
+        modifiers: {
+            global: ["is-selected"],
+        },
+        children: {
+            default: `block link`,
+        },
+        template: blockLinkTemplate,
+    });
+
+    // Base + danger
+    runComponentTests({
+        type: "visual",
+        baseClass: "s-block-link",
+        modifiers: {
+            primary: ["danger"],
+        },
```

Comment:
This should be a variant.

Reply:
Nvm, for the purposes of this test, primary works better to generate all variations. Keeping it as-is.